neuromation/cli/utils.py

```diff
@@ -461,11 +461,6 @@ def volume_to_verbose_str(volume: Volume) -> str:
             f"{details} to a job-ID: {e}"
         )
     if jobs:
-        if len(jobs) > 1:
-            log.warning(
-                f"Found {len(jobs)} jobs matching {details}: "
-                ", ".join(job.id for job in jobs)
-            )
         job_id = jobs[-1].id
```

Comment:
Would not be better to return always *your* job if there is such in the found jobs and return the last job only if no of them are your?

Reply:
Ah, the owner is always specified.

Reply:
Yes, as we agreed in https://github.com/neuromation/platform-client-python/issues/1024, `neuro kill job-name` will also pass the parameter `owner=myself` when resolving `job-name` to job-ID